plugin/src/main/java/com/exadel/aem/toolkit/plugin/handlers/common/MaxChildrenHandler.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.plugin.handlers.common;
+
+import com.exadel.aem.toolkit.api.annotations.meta.Scopes;
+import com.exadel.aem.toolkit.api.annotations.policies.MaxChildren;
+import com.exadel.aem.toolkit.api.annotations.policies.PolicyTarget;
+import com.exadel.aem.toolkit.api.handlers.Handler;
+import com.exadel.aem.toolkit.api.handlers.Source;
+import com.exadel.aem.toolkit.api.handlers.Target;
+import com.exadel.aem.toolkit.plugin.utils.DialogConstants;
+
+/**
+ * Implements {@code BiConsumer} to populate a {@link Target} instance with properties originating from a {@link Source}
+ * object. The source refers to the {@code resolvemaxchildren} listener of the {@code
+ * cq:editConfig} node of an AEM component
+ */
+public class MaxChildrenHandler implements Handler {
+    /**
+     * Name of the listener to resolve max children limit for `policies` js module
+     */
+    public static final String MAX_LIMIT_RESOLVER_NAME = "resolvemaxchildren";
```

Comment:
Probably, `MAX_CHILDREN` or just `LIMIT`, not "max limit")
The same in the Javadoc comments below. The word "max" already includes the meaning "limit". If something is "max", no one can go beyond this value. Therefore, this value is by def a limit...